docs/modules/operation/pages/deep-dive/provisioning/foreign-source.adoc

```diff
@@ -0,0 +1,53 @@
+
+[[foreign-source-definition]]
+= Foreign Source Definition
+
+The default foreign source definition serves as a template that defines the services to detect on (DNS, FTP, ICMP, and so on), the scan interval for discovery, and the policies to use when provisioning.
+
+Policies determine entity persistence and/or set attributes on the discovered entities that control {page-component-title} management behavior.
+Provisiond applies the existing default foreign source definition unless you choose to modify it.
+
+This section describes how to edit the default foreign source definition.
+
+== Edit detectors and set scan interval
+
+. In the web UI, click the *Gear* symbol in the top-right of the screen.
+. Choose menu:Provisioning[Manage Provisioning Requisitions > Edit Default FS].
++
+image::provisioning/FS_def.png[]
++
+NOTE: The screen displays the list of service detectors and a tab to view and define policies.
+Provisiond scans the services in the order in which the detectors appear in the list.
+
+** Click the *Edit* symbol to edit a service detector's parameters.
+To add more parameters (retries, timeout, port, and so on), click *Add Parameter* and define its name and value.
+** Click the *Move* symbol to move a service detector.
+** Click the *Delete* symbol to delete a service detector.
+** To undo a change, click *Reset to Default* at the top-right of the screen.
+. Click *Save*.
+. (Optional) Update the scan interval using one of the following:
++
+* w: weeks
+* d: days
+* h: hours
+* m: minutes
+* s: seconds
+* ms: milliseconds
++
+For example, to prompt a scan every 6 days and 53 minutes, use `6d53m`.
+Type `0` to disable automatic scanning.
+
+. Click *Save* to save the foreign source definition.
+
+== Create policies
+
+. In the web UI, click the *Gear* symbol in the top-right of the screen.
+. Choose menu:Provisioning[Manage Provisioning Requisitions > Edit Default FS] and click the Policies tab.
+. Click *Add Policy*, type a name, select the class from the *Class* list, and enter the required information.
+** Type a space in any of the lists to see their options.
+. (Optional) Click *Add Parameter* to add more parameters to the class (retries, timeout, port, and so on).
+. Click *Save*.
+. Repeat for any additional policies you want to add.
+. Click *Save* to save the foreign source definition.
+
+NOTE: To return to the default foreign source definition, click *Reset to Default*.
```

Comment:
```suggestion
NOTE: To return to the default foreign source definition, click *Reset to Default*.
This will reset any changes that you have made to the definition.
```